user-guide/Getting_started/Accessing_DataMiner/Accessing_DataMiner_Cube/Opening_DataMiner_Cube/Options_for_opening_DataMiner_Cube.md

```diff
@@ -76,15 +76,31 @@ For example:
 
 ### alarm=
 
-Use this option to immediately display a specific alarm card.
+Use this option to immediately display a specific alarm card. Different formats are supported:
 
-Specify the DMA ID and root alarm ID.
+- DMA ID / root alarm ID
 
-Example:
+  Example:
 
-```txt
-http://MyDMA1/DataMinerCube/DataMinerCube.xbap?alarm=48/6713
-```
+  ```txt
+  alarm=48/6713
+  ```
+
+- DMA ID / element ID / root alarm ID
```

Comment:
I've checked this on my DMA (using 10.4.3), and this format didn't work. Does this require a specific DataMiner version? Could you double-check?

Reply:
10.4.4 will be released on Friday, March 22nd and this format will already be available.

Reply:
Do you mean this will be included in 10.4.4? Which release note adds this?

Reply:
Exactly.
https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=39126

Reply:
As this is a release note for the Swarming soft-launch feature, do users need to have [the Swarming soft-launch option](https://docs.dataminer.services/user-guide/Reference/Soft-launch_options/Overview_of_Soft_Launch_Options.html#swarming) enabled to be able to use this format? (Also, note that the version in the RN is 10.4.5, not 10.4.4.)

Reply:
The new alarm IDs format will be utilized regardless of whether the Swarming soft-launch option is enabled or not. It offers a more precise description of alarm locations. Users are encouraged to opt for versions supporting the new format over the old one.
The changes will indeed be implemented in version 10.4.5 as stated in the Release Notes.